doc/src/manual/methods.md

```diff
@@ -335,7 +332,17 @@ Here's an example where the method type parameter `T` is used as the type parame
 type `Vector{T}` in the method signature:
 
 ```jldoctest
-julia> myappend(v::Vector{T}, x::T) where {T} = [v..., x]
+julia> function myappend(v::Vector{T}, x::T) where {T}
+           return [v..., x]
+       end
+myappend (generic function with 1 method)
+```
+
+The type parameter `T` in the function's signature ensures that the type of the new `v` is preserved after element `x` is appended to it. 
+The function's signature is preceded by the keyword `where` and is placed immediately after the method arguments. When only one type parameter is used, the curly braces are optional. An equivalent one-line definition is:
```

Comment:
```suggestion
The function's signature is preceded by the keyword `where` and is placed immediately after the method arguments. When only one type parameter is used, the curly braces are optional. An equivalent one-line definition would be:
```

Reply:
I would just delete the part about the one-line definition. The two equivalent ways of defining methods are described at the top of the [Functions](https://docs.julialang.org/en/v1/manual/functions/) section of the manual, I don’t see the point of repeating it here.

Reply:
I am fine with either way.

Reply:
sostock, you're right of course, but while I knew about the one-line style of definition I still got confused because my brain was focused on trying to understand the notation for types. And the section on Functions does not show types. I think it's useful to verbally explain the meaning of `f(x::T) where T = T`. The explanation could be moved a few lines down where this appears: `mytypeof(x::T) where {T} = T`.

Reply:
Maybe the one-line part could just be shortened, basically just saying something like “parametric method definition also works with the short-form syntax, which would look like this:”

Reply:
> This is my first PR and I'm not entirely sure how I was to proceed. I went to the "commits" tab, edited, and committed again. I didn't want to overwrite the suggestions made by Simeon though. But do feel free to edit and put this matter to rest. Many thanks for your help!